backend/schema/corpora_schema.md

```diff
@@ -1,125 +1,68 @@
 
 *   **Unique observation identifiers**. Each observation (usually a cell) must have an id that is unique within the dataset. This is usually already
     present in every submission, for example as a barcode.
 *   **Unique feature identifiers**. Every feature (usually a gene or transcript) also needs a unique identifier. This is occasionally not present because
-    of one-to-many mappings between gene symbols and other gene ids. In cases where there are duplicated feature identifiers, they will be summed during
-    curation.
-*   **Type annotations or inferrable types**. For both expression values and metadata, we don't want to have to guess at the type. Typing is already explicit
-    in anndata, loom, Seurat and Bioconductor formats.
+    of one-to-many mappings between gene symbols and other gene ids. In cases where there are duplicated feature identifiers, they will need to be appropriately
+    combined before submission.
 *   **No PII**. No metadata can be personally identifiable: no names, dates of birth, specific locations, etc. There's a
-    [list](https://docs.google.com/document/d/1nlUuRiQ8Awo_QsiVFi8OWdhGONjeaXE32z5GblVCfcI/edit?usp=sharing).
+    [list](https://docs.google.com/document/d/1nlUuRiQ8Awo_QsiVFi8OWdhGONjeaXE32z5GblVCfcI/edit?usp=sharing). It is not always possible to determine
+    automatically whether metadata contains PII. For example, a field that looks like a date could be a date of birth or just some other date. So tooling will
+    warn submitters for potential PII and allow them to attest that it is not personally identifiable.
 
-
-#### Matrix Layers
+### Matrix Layers
 
 The count matrix itself can exist in several forms. Generally, there is a "raw" count matrix without scaling, filtering, normalization, etc. And there is
 a "final" matrix that is used in the publication with QC and various corrections. There can also be several intermediate matrices.
 
-These different transformations of the count matrix are called "layers" in several file formats. Submissions to Corpora must contain at least one layer,
+These different transformations of the count matrix are called "layers" in several file formats. Submissions to the Data Portal must contain at least one layer,
 the "raw" layer. They may optionally also have a "final" layer used for presentation, and each layer must be identified.
 
 
-### Remix Metadata
-
-The remaining part of the schema defines what's needed for the Corpora remix. The words "must" and "requirement" are used throughout, but note the
-discussion above regarding curation policies as well the [caution against](#Software-vs-policy) introducing any
-software dependencies on these fields.
-
-
-#### Schema Version
+### Schema Version
 
-Remixed Corpora datasets must store the version of the schema they follow (that is, the version of this document) as
+Datasets in the Data Portal must store the version of the schema they follow (that is, the version of this document) as
```

Comment:
What does the `corpora_encoding_version` track?

Reply:
It's the version of the encoding doc. I'll add a link and explanation.